packages/DateTimePickerCommon/src/CustomPopper.tsx

```diff
@@ -21,6 +22,7 @@ export const CustomPopper = ({
 const StyledCustomPopper = styled.div(
   ({ popperStyles }: { popperStyles: CSSObject }) => css`
     ${datePickerStyles};
+    ${FixAriaMessageStyle};
```

Comment:
doesn't work if you apply the style just right here?
I think we should not use a global style in this case because we can style the `Popper` directly

Reply:
It works only for the `DatePicker` because the aria message is displayed inside the `Popper`. 
But for the`TimePicker`, it doesn't work because the message is displayed above the input:
![Screenshot 2023-11-07 at 14 46 19](https://github.com/WTTJ/welcome-ui/assets/1446004/84fcca81-738b-4e43-93f4-bc0dd6476bf1)

Reply:
maybe we can use this solution for the `DatePicker` and for the `TimePicker`, use a wrapper that includes this style?

Reply:
✅